TypeScript/publish.ts

```diff
@@ -55,99 +48,81 @@ export function publish(
 				options as unknown as ValidPublishOptions;
 
 			function c(e: unknown) {
-				console.error("publish command didnt work for", module.name!);
+				console.error("publish command didnt work for", module.name);
 				console.error(e);
 			}
-			try {
-				const commandData = {
-					type: CommandTypeRaw[module.type],
-					name: module.name!,
-					description: [CommandType.Slash, CommandType.Both].includes(
-						module.type
-					)
-						? module.description
-						: undefined,
-					options: [CommandType.Slash, CommandType.Both].includes(
-						module.type
-					)
-						? optionsTransformer(
-								(module as SlashCommand).options ?? []
-						  )
-						: [],
+			const log = (...message : any[]) => () => console.log(...message);
+			const logged = (...message : any[]) => log(message);
+			const appCmd = <V extends CommandType, S, T>(t: V) => {
+				return (is: S, els: T) => (t & CommandType.Both) !== 0
+					? is
+					: els
+			}
```

Comment:
add some comments explaining what it does, because eventually it'll be in someone's project